app/src/main/java/com/github/onlynotesswent/ui/common/FlashcardComposables.kt

```diff
@@ -519,14 +523,222 @@ fun FlashcardItemDropdownMenu(
+              .testTag("flashcard")
+              .padding(5.dp)
+              .graphicsLayer {
+                rotationY =
+                    when (flipState.value) {
+                      FlipState.FRONT -> 360f - rotation.value // same rotation for both sides
+                      FlipState.BACK -> rotation.value
+                    }
+                cameraDistance = 15 * density
+              }
+              .clickable { flipState.value = flipState.value.next }) {
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.SpaceEvenly,
+            modifier =
+                Modifier.fillMaxWidth()
+                    .padding(vertical = 30.dp)
+                    .heightIn(min = 200.dp, max = 400.dp)) {
+              if (rotation.value <= 90f || rotation.value >= 270f) {
+                Text(
+                    flashcard.value.front,
+                    style = Typography.bodyLarge,
+                    fontWeight = FontWeight(450),
+                    modifier = Modifier.testTag("flashcardFront"))
+                FlashcardImage(flashcard, fileViewModel)
+              } else {
+                Text(
+                    flashcard.value.back,
+                    style = Typography.bodyLarge,
+                    fontWeight = FontWeight(450),
+                    modifier = Modifier.testTag("flashcardBack").graphicsLayer { rotationY = 180f })
+              }
+            }
+      }
+}
+
+@Composable
+fun McqPlayItem(
+    flashcard: State<Flashcard>,
+    fileViewModel: FileViewModel,
+    onCorrect: () -> Unit,
+    onIncorrect: () -> Unit,
+    choice: MutableState<Int?> = remember { mutableStateOf(null) }
+) {
+  val backs =
+      listOf(flashcard.value.back) +
+          flashcard.value.fakeBacks.filter { it != flashcard.value.back && it.isNotBlank() }
+  val shuffledIndexes = backs.indices.shuffled()
+
+  ElevatedCard(modifier = Modifier.fillMaxWidth(0.9f).padding(5.dp).testTag("flashcard")) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.spacedBy(20.dp),
+        modifier =
+            Modifier.fillMaxWidth()
+                .padding(vertical = 30.dp, horizontal = 10.dp)
+                .heightIn(min = 200.dp, max = 600.dp)) {
+          Text(
+              flashcard.value.front,
+              style = Typography.bodyLarge,
+              fontWeight = FontWeight(550),
+              modifier = Modifier.testTag("flashcardFront"))
+          FlashcardImage(flashcard, fileViewModel)
+          HorizontalDivider(modifier = Modifier.height(5.dp).fillMaxWidth().padding(8.dp))
+          shuffledIndexes.forEach { index ->
+            val color = remember { mutableStateOf(Color.Gray) }
+            color.value =
+                if (choice.value != null && index == 0) {
+                  Color.Green.copy(green = 0.6f, blue = 0.3f, red = 0.3f)
```

Comment:
Is there a way to use the app them color? Like the Danger color instead of red for exemple

Reply:
We will handle this in the color overhaul of the app